appengine/standard/ndb/schema_update/display_entities_handler.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import jinja2
+import webapp2
+
+import models_v1
+import models_v2
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__)),
+    extensions=['jinja2.ext.autoescape'],
+    autoescape=True)
+
+
+def get_current_entities(updated_schema=False):
+    if updated_schema:
+        reload(models_v2)
+        current_entities = list(models_v2.Picture.query().fetch())
+    else:
+        reload(models_v1)
+        current_entities = list(models_v1.Picture.query().fetch())
+    return current_entities
+
+
+class DisplayEntitiesHandler(webapp2.RequestHandler):
+    def get(self):
+        updated_schema = self.request.params.get('updated')
+        if updated_schema is None:
```

Comment:
No need for this. `None` is falsy in Python, and if updated is set to any non-zero-length string it'll be truthy. https://docs.python.org/2.4/lib/truth.html